### PR TITLE
Added C# Sample Code

### LINKED ISSUE #440: Add Sample Code for C# in CONTRIBUTING.md
#### This issue is about updating Sample Codes.

- [ ] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #440 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [ ] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -156,3 +156,28 @@ function main () {
 
 main();
 ```
+
+#### CSharp
+
+```csharp
+public class QuickSort
```
Comment (mohitkyadav) [inline CONTRIBUTING.md]: Please also add `using` statement.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,7 +32,7 @@
 * Filename should be derived from the folder name. (eg `longest_common_subsequence` becomes `longestCommonSubsequence.c` or `LongestCommonSubsequence.java`)
 * Name of master function of the code should be kept same as filename to the best extent possible.
 * Prefer classes instead of multiple helper functions (where applicable).
-* Currently we are accepting contributions in C, C++, Java, Python, Go and JavaScript but other languages may be considered after a discussion.
+* Currently we are accepting contributions in C, C++, CSharp, Java, Python, Go and JavaScript but other languages may be considered after a discussion.
```
Comment (Anshuman-Verma) [inline CONTRIBUTING.md]: Why not use `C#` ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -156,3 +156,30 @@ function main () {
 
 main();
 ```
+
+#### CSharp
```
Comment (aashutoshrathi) [inline CONTRIBUTING.md]: Should I make it C# too ?

Comment (Anshuman-Verma) [inline CONTRIBUTING.md]: Probably that should be for the best ! 👍

**Pull request merged**